tests/main/appstream-id/task.yaml

```diff
@@ -1,28 +1,34 @@
 summary: Verify AppStream ID integration
 
-# fedora-*, amazon-*, centos-*: use nmap netcat by default (https://nmap.org/ncat/)
-systems: [-fedora-*, -amazon-*, -centos-*]
+# ubuntu-core: no curl
+# ubuntu-14.04: curl too old, missing --unix-socket
+systems: [-ubuntu-14.04-*, -ubuntu-core*]
 
 prepare: |
     snap install jq
 
+debug: |
+    if [ -e response ]; then
+        grep -n '' response
+    fi
+
```

Comment:
response file can be removed afterwards?

Reply:
Added the cleanup. Although, IIRC @sergiocazzolato added something to clean up the test tree automatically on each prepare/restore. Isn't that right?

Reply:
Yes, the test dir is cleaned everytime iirc